example/lib/main.dart

```diff
@@ -19,9 +19,10 @@ class MyApp extends StatefulWidget {
 
 class _MyAppState extends State<MyApp> {
   @override
-  void initState() {
+  void initState()  {
```

Comment:
I think there's an extra blank space that could be removed. You can run the format style tool here (`Cmd+Alt+L`)